.github/workflows/enos-verify-stable.yml

```diff
@@ -0,0 +1,17 @@
+name: enos-verify-stable
```

Comment:
Am I correct in assuming that this workflow is triggered by CRT when something is promoted between "unstable" and "stable", i.e. after a push to main and the build workflow is run? If that's correct, we're doing almost the same validation of the same artifacts twice, except that validation is happening before and after security scanning, notorization, etc. Perhaps we ought not execute the enos-run scenario on pushes to main as we're now hooking into the pipeline at a later (better) step in the pipeline.

Reply:
This is triggered after the verify artifact stage and runs Enos scenarios on the artifact in `stable` .  The scenario matrix run with this, tests more variants than the one run on PR.  We can add or reduce the number of scenarios to test based on this iteration.  After the artifact in stable is validated against these scenarios it should be ready to promote.  Let me know if this looks good to you as the first iteration of it.